src/components/footer.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import logoAdalab from "../images/logo-adalab-80px.png";
+import "../stylesheets/scss/main.scss";
+
+function Footer() {
+    return (
+        <footer className="footer"> 
+        <small className="footer_small"> Awesome profile-cards @ Sense.JS 2019</small>
```

Comment:
Y en el fichero del componente pondríamos las props. Creo que es un componente estado estado (no estoy muy segura que lo sea), entonces creo que iria con clase: 
class Footer extends React.Component
y las props estarían precedidas de this